sentry/src/main/java/io/sentry/EventProcessor.java

```diff
@@ -1,9 +1,17 @@
 package io.sentry;
 
+import io.sentry.protocol.SentryTransaction;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 public interface EventProcessor {
   @Nullable
-  SentryEvent process(@NotNull SentryEvent event, @Nullable Object hint);
+  default SentryEvent process(@NotNull SentryEvent event, @Nullable Object hint) {
```

Comment:
As much as I understand and agree with the implementation, I think this interface is less obvious now since it can process both events and transactions and both have default implementation - Javadoc explaining the purpose and what does it do would be nice.

Reply:
makes sense